src/chapters/11-route-params.md

```diff
@@ -95,7 +110,19 @@ Alright, we have just one more step left here: updating the tests. We can add an
 git add tests/integration/components/rental-test.js
 ```
 
-Awesome! However, there is one more point to mention here: by default, a component test (like the one that we have here) does not set up the router. Usually, this is what you'd want, since for more components, you likely aren't going need the router to test what you want. But in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
+If we run the tests in the browser, everything should...
+
+<!-- TODO: Make sure we show screenshot of failing test here. Otherwise reader might be confused that their tests are failing before they read the explanation below. -->
+```run:screenshot width=1024 height=768 retina=true filename=fail.png alt="The test failed"
+visit http://localhost:4200/tests?nocontainer&deterministic
+wait  #qunit-banner.qunit-fail
+```
+
+...wait a minute, it didn't work!
+
+About time something didn't Just Work™ the first time we tried; this is the *advanced* part of the tutorial after all.
+
+There is one more point to mention here: by default, a component test (like the one that we have here) does not set up the router. Usually, this is what you'd want, since for most components, you likely aren't going to need the router to test what you want. But in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
```

Comment:
```suggestion
By default, a component test (like the one that we have here) does not set up the router. Usually, this is acceptable, but in this specific case, we have a `<LinkTo>` in our component that is relying on the router to generate its URLs.
```

Reply:
Now that I'm reading this, feels weird to go from "it didn't work" to "by default this is how component tests work". Need a sentence here that bridges the gap between the two.